app/routes/event/$eventId.tsx

```diff
@@ -18,9 +18,13 @@ export const loader: LoaderFunction = async ({ request }) => {
   const url = new URL(request.url);
   const cookie = request.headers.get('cookie');
 
+  const eventId = url.pathname.split('/')[2];
+
+  if (eventId === 'new') return { event: dummyEvent };
```

Comment:
will not dummy event contain a fixed date, when user clicks on new, should we change it to current date?

Reply:
@isVivek99 handled that in `useEffect`. We cannot get the navState in loader function of remix, otherwise I would have used it here itself.